tasks/main.yml

```diff
@@ -23,6 +23,14 @@
     - "{{ spark_log_dir }}"
     - "{{ spark_run_dir }}"
 
+- name: Ensure Spark download and install directories exist
```

Comment:
Does it make sense to append `spark_src_dir` and `spark_usr_parent_dir` to the task above this, or is the idea to keep these two directories so that `owner` and `group` aren't owned by `spark`?